src/trade_aggregation_call_builder.js

```diff
@@ -0,0 +1,66 @@
+import { CallBuilder } from "./call_builder";
+import { NotFoundError, NetworkError, BadRequestError } from "./errors";
+
+/**
+ * Trade Aggregations facilitate efficient gathering of historical trade data
+ * Do not create this object directly, use {@link Server#tradeAggregation}.
+ * @param {string} serverUrl serverUrl Horizon server URL.
+ * @param {Asset} base base asset
+ * @param {Asset} counter counter asset
+ * @param {long} start_time lower time boundary represented as millis since epoch
+ * @param {long} end_time upper time boundary represented as millis since epoch
+ * @param {long} resolution segment duration as millis since epoch. *Supported values are 5 minutes (300000), 15 minutes (900000), 1 hour (3600000), 1 day (86400000) and 1 week (604800000).
+ * @returns {OrderbookCallBuilder}
+ */
+export class TradeAggregationCallBuilder extends CallBuilder {
+  constructor (serverUrl, base, counter, start_time, end_time, resolution){
+    super(serverUrl);
+    this.allowedResolutions = [300000, 900000, 3600000, 86400000, 604800000];
```

Comment:
I'd put this as a `const` out of the scope of the class (above it).  Also, you can use ES6 [Sets](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) which makes it super easy to query. 
```javascript
const allowedResolutions = new Set([300000, 900000, 3600000, 86400000, 604800000])
```

Reply:
Thanks. Didn't know about sets.. will check it out.